util/src/main/java/tech/pegasys/pantheon/util/bytes/BytesValue.java

```diff
@@ -139,6 +139,21 @@ static BytesValue wrapBuffer(final Buffer buffer) {
     return wrapBuffer(buffer, 0, buffer.length());
   }
 
+  /**
+   * Wraps a full Vert.x {@link Buffer} as a {@link BytesValue}.
+   *
+   * <p>Note that as the buffer is wrapped, any change to the content of that buffer may be
+   * reflected in the returned value.
+   *
+   * @param buffer The buffer to wrap.
```

Comment:
this is a useless comment, but what about "`The {@link Buffer} to wrap.`"

Reply:
Fair point, but I think I'll just leave it as-is so that the javadoc across this class is consistent.